Pixel Game/Assets/Scripts/DontDestroyTestScripts/CMvcam1DontDestroy.cs

```diff
@@ -1,12 +1,24 @@
-﻿using System.Collections;
+﻿using Cinemachine;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.SceneManagement;
 
 public class CMvcam1DontDestroy : MonoBehaviour
 {
     private static bool cMvcam1Exists;
+    private GameObject player;
+    private CinemachineVirtualCamera cinemachineVirtualCamera;
+    private string nameOfMainMenu = "Main_Menu_Lasse";

```

Comment:
This feels like it should be a constant (and be named with initial upper case to follow C# naming convention for constants):
` private const string NameOfMainMenu = "Main_Menu_Lasse";`

Reply:
A const is just declaring that this variable will NOT change am i correct?

Reply:
Fixed, btw ^^

Reply:
Yes, correct